aten/src/ATen/native/TensorIterator.h

```diff
@@ -332,7 +343,7 @@ struct CAFFE2_API TensorIterator {
   bool accumulate_ = false;
   bool resize_outputs_ = true;
   bool is_reduction_ = false;
-  bool compute_common_dtype_ = true;
+  CommonDTypeStrategy compute_common_dtype_strategy_ = CommonDTypeStrategy::COMPUTE_ALL;
```

Comment:
You should move this to under the `int`. This prevents the bools from getting packed by the compiler into one word.